gtsfm/visualization/view_scene.py

```diff
@@ -60,7 +60,7 @@ def view_scene(args: argparse.Namespace) -> None:
     mean_pt = compute_point_cloud_center_robust(point_cloud)
 
     # Zero-center the point cloud (about estimated center).
-    zcwTw = Pose3(Rot3(np.eye(3)), -mean_pt)
+    zcwTw = Pose3(Rot3(), -mean_pt)
```

Comment:
im not sure if this was Rot3 or Point3 but I remember one of them not zero-initializing when constructed. this was a long time ago.

Reply:
It’s Point3